provider/cmd/pulumi-resource-tls/schema.json

```diff
@@ -604,12 +307,12 @@
                 },
                 "caKeyAlgorithm": {
                     "type": "string",
-                    "description": "Name of the algorithm used when generating the private key provided in `ca_private_key_pem`. **NOTE**: this is deprecated and ignored, as the key algorithm is now inferred from the key.\n",
-                    "deprecationMessage": "This is now ignored, as the key algorithm is inferred from the `ca_private_key_pem`."
```

Comment:
The deprecation message is gone from [upstream](https://github.com/t0yv0/terraform-provider-tls/blob/1239aabb49af41a01fd4ec76b6d13303f2083499/internal/provider/resource_locally_signed_cert.go#L184-#L184).